### ISSUE TITLE
How to define static class variable?

### ISSUE THREAD

**Issue by 2xj891s** [cython/cython#2115]
How to define static class variable and constant in Cython just as following code's semantic?
```
1:  cdef class Car:
2:      cdef int battery=100 # Instance variable
3:      cdef static int total_car_count = 0 # Static variable shared among classes.
4:      cdef const int WHEEL_COUNT = 4 # Constant of class Car.
```

As far as I know, the syntax of `line 3` and `line4` is not supported by Cython..
But is there any syntax to define static and constant variables in Cython?
If not, is there any way to mimic the semantics of static and constant variables?

Thanks

**Comment by robertwb** [comment]
Correct, this is not yet supported. For now, the best way to do this is
simply to use a module-level global (and possibly do an assignment to a
default value in your __cinit__ method).

**Comment by scoder** [comment]
Also see #575.
Note that these values would always have to be defined in the declaration, i.e. in the `.pxd` file if there is one.

**Comment by FilipDepta** [comment]
A few years have passed since the last comment. Did anything change?

**Comment by FilipDepta** [comment]
Ok, thank you. In my case, the class variable will be used in heavy number crunching, so a better option is to define it as a global C variable, I think.
